src/client/components/dimension-tile/dimension-tile.tsx

```diff
@@ -174,7 +176,10 @@ export class DimensionTile extends React.Component<DimensionTileProps, Dimension
     }
 
     if (sortOn.reference instanceof Measure) {
-      query = query.performAction(sortOn.reference.toApplyExpression(0));
+      const measure = sortOn.reference;
+      const series = new ConcreteSeries(MeasureSeries.fromMeasure(measure), measure);
+      // TODO: FIX that
+      query = query.performAction(series.plywoodExpression(0, MeasureDerivation.CURRENT, null));
```

Comment:
looks bad, but we need ad hoc sort expression; maybe encapsulate that in `SortOn`